### PR TITLE
Feature/manual subscription renewal

### PR DESCRIPTION
**Pull request by rhimmelbauer**
Notes:

Easy way to renew a subscription to a profile.

### CODE REVIEW

File: vendor/templates/vendor/manage/profile_detail.html [hunk 1]
```diff
@@ -80,11 +80,11 @@ <h2>{% trans 'Admin: Profile Details' %}</h2>
       </div>
     </div>
   </div>
-  <div class='col-6'>
+  <div class='col-4'>
     <div class='card'>
       <div class='card-header'>
         <div class='d-flex flex-row justify-content-between'>
-          {% trans 'Products Owned' %}
+          {% trans 'Ative Products Owned' %}
```
Comment (mgcamisa) [inline vendor/templates/vendor/manage/profile_detail.html:87]: typo: should be "Active"

File: vendor/templates/vendor/manage/profile_detail.html [hunk 2]
```diff
@@ -115,7 +118,45 @@ <h2>{% trans 'Admin: Profile Details' %}</h2>
       </div>
     </div>
   </div>
-  <div class="col-6">
+  <div class='col-4'>
+    <div class='card'>
+      <div class='card-header'>
+        <div class='d-flex flex-row justify-content-between'>
+          {% trans 'Inative Products Owned' %}
```
Comment (mgcamisa) [inline vendor/templates/vendor/manage/profile_detail.html:125]: typo: should be "inactive"

File: vendor/templates/vendor/manage/profile_detail.html (hunk 1 above)
Comment (rhimmelbauer) [reply vendor/templates/vendor/manage/profile_detail.html]: thanks fixed

File: vendor/templates/vendor/manage/profile_detail.html (hunk 2 above)
Comment (rhimmelbauer) [reply vendor/templates/vendor/manage/profile_detail.html]: thanks fixed

**Review by mgcamisa — APPROVED** [review]

**Pull request merged**